corehq/apps/app_manager/static/app_manager/js/app_summary.ng.js

```diff
@@ -187,6 +190,32 @@
             return utils.translateName(form_module.name, $scope.lang);
         };
 
+        $scope.getModuleFormIcon = function(form_module) {
+            var formIcon = 'fa fa-file-o appnav-primary-icon';
+            if (form_module.action_type === 'open') {
+                formIcon = 'fcc fcc-app-createform appnav-primary-icon appnav-primary-icon-lg';
+            } else if (form_module.action_type === 'close') {
+                formIcon = 'fcc fcc-app-completeform appnav-primary-icon appnav-primary-icon-lg';
+            } else if (form_module.action_type === 'update') {
+                formIcon = 'fcc fcc-app-updateform appnav-primary-icon appnav-primary-icon-lg';
+            };
+            return formIcon;
+        }
+
+        $scope.getModuleIcon = function(form_module) {
```

Comment:
Feel free to reject as out of scope, but it would be nice to pull this from the server, so changes don't have to update both here and `module_link_primary.html`.